src/app/ui/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,66 @@
+import { forwardRef } from 'react';
+
+import * as RadixTabs from '@radix-ui/react-tabs';
+import { css } from 'leather-styles/css';
+
+const tabRootStyles = css({
```

Comment:
Can we always pull the styles out into a separate `.styles.ts` file?

Reply:
What's the reason for this ask? The pattern of style props is that we couples styles to the component. Moving styles to a separate file goes against that no?

Reply:
I have often seen style props in their own styles file just to break up files and make them easy to read. I suppose just a preference.

Reply:
I just worked on a project before that did this, which seemed to scale better, but not critical to change.

Reply:
Two thoughts on this mostly, and would love to get consistency but think root problem is we're taking different approaches to components. Discussing on Slack.

Unlike the other components, here all I'm doing is adding the styles, so if I move styles to another file, that's basically everything?

I also used to separate styles from everything, when working on SCSS based projects. Since moving to React, we do css-in-js where the pattern is to couple them with the components. Moving away from this is a paradigm we've never followed, so curious why only now we'd want to start doing this?

Reply:
I also like having them separate if there's a lot of styles but I don't think it's a big deal. 

Here we could probably have separate files for `Root`, `Trigger`, `List` that have just those components and their styles.

Doing that though and we have the overhead of loads of files to manage so I don't think it matters much.

Reply:
I'm a big proponent of keeping styles separate from the general rest of the components, and push for the `.layout.ts` structure. Does this pattern not take it further in having `.layout.ts` and `styles.tsx`? What's the benefit of splitting apart these two?

Reply:
I think it's slick to have modular files where you have style + JS for a component in one small file. 

So you'd have:
```
const tabRootStyles = css({
....

export function TabRoot..
```
That's how Svelte works also. Everything is a self contained component managing it's own markup and style

Often you may have repeated styles though at which point it's nice to abstract them into objects in a shared styles file.

Reply:
Based on what we merged and [Fara's last PR](https://github.com/leather-wallet/extension/pull/4799) we went for:
- keeping styles in the same file as components
- having a bigger component file with more sub-components in there rather than loads of files

I'm happy with that and think it will work nicely